### ISSUE TITLE
Object Browser Description

### ISSUE THREAD

**Issue by dimitrimitropulos** [rubberduck-vba/Rubberduck#1523]
I noticed that nothing in `Rubberduck` or `Rubberduck_SourceControl` has Descriptions in the Object Browser.  Definitely not super necessary, but it might be slick to include them.

To be clear, I'm talking about the description you can set (among other ways?) with 
`Attribute SomeFunction.VB_Description = "my description"`.

Might be a slick finishing touch at some point.

**Comment by PeterMTaylor** [comment]
To ask and clarify @dimitrimitropulos  isn't RD when loaded in the VBE show their objects/references and U could see their items?

**Comment by dimitrimitropulos** [comment]
Yep! You can see the items, but there is no description.  [Example](https://christopherjmcclellan.wordpress.com/2015/04/21/vb-attributes-what-are-they-and-why-should-we-use-them/)

**Comment by retailcoder** [comment]
This is fairly simple, adding `[Description("description here")]` attributes to the types (interfaces?) involved should do the trick.

The descriptions to use for the `IAssert` interface are in the wiki; I'll take any well-worded description for everything else.
